BLSQ/iaso
# IA-3306: Add "Projects" card on landing page

Explain what problem this PR is resolving

Related JIRA tickets : [IA-3306](https://bluesquare.atlassian.net/browse/IA-3306)

## Self proofreading checklist



## Changes

Add "Projects" card on landing page.

## How to test

From setuper folder, launch `python3 setuper.py` to create data and login with the created account.
Then check the home page: http://localhost:8081/dashboard/home/.

## Print screen / video

![Iaso-Landing-page](https://github.com/user-attachments/assets/085c3e76-54f6-4d5c-a3a7-e850126a681e)



[IA-3306]: https://bluesquare.atlassian.net/browse/IA-3306?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
